src/gc.h

```diff
@@ -4,12 +4,15 @@
 
 #include "redismodule.h"
 #include "util/dllist.h"
+#include "dep/thpool/thpool.h"
 #include <time.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+extern threadpool gcThreadPools_g;
```

Comment:
why extern and not static? is this accessed outside gc.c?

Reply:
We made it configurable with FORK_GC_THREAD_POOL_SIZE flag

Reply:
Yeah, but the symbol itself is not accessed outside gc.c; `static` doesn't mean "not relevant to anything else in the code", it simply means whether other compilation units (=files) can have direct access to that symbol.